templates/kubeflow/what-is-kubeflow.html

```diff
@@ -512,14 +511,18 @@ <h2 class="u-fixed-width u-sv1">Who uses Kubeflow?</h2>
     </div>
     <div class="col-8">
       <h2 class="u-fixed-width u-sv1">Get started today</h2>
-      <p class="u-sv2">Try-out Kubeflow on MicroK8s - Zero-ops Kubernetes with high availability. Single-command deploy locally on your desktop, public cloud VM or on-prem server.</p>
-      <a class="p-button--positive" href="/kubeflow/install">
-      Try Kubeflow
-      </a>
+      <p class="u-sv2 p-heading--four">Try-out Kubeflow on your K8s deployment. Or on MicroK8s - Zero-ops Kubernetes with high availability.</p>
+      <p>Single-command deploy locally on your desktop, public cloud VM or on-prem server.</p>
+      <p>
+        <a class="p-button--positive p-link--external" href="https://charmed-kubeflow.io/docs/install">
+        Try Kubeflow
+        </a>
+        <a class="p-link--external" href="https://charmed-kubeflow.io">
```

Comment:
In the doc this link is: 
```suggestion
        <a class="p-link--external" href="https://microk8s.io/docs/addon-kubeflow">
```